src/java/org/apache/cassandra/service/consensus/migration/ConsensusMigrationMutationHelper.java

```diff
@@ -262,17 +266,18 @@ public static IAccordResult<TxnResult> mutateWithAccordAsync(ClusterMetadata cm,
         long minEpoch = Epoch.EMPTY.getEpoch();
         for (IMutation mutation : mutations)
         {
+            boolean userTimestamp = mutation instanceof Mutation && ((Mutation) mutation).isUserTimestamp();
```

Comment:
Again, this isn't a useful granularity. In practice mutation is not a grouping upon which `userTimestamp` is set its per row?

We don't need this intermediate step that only works with some CQL but not others and adds a lot of noise and needs to be replaced anyways.

Just the flag to `mutateWithTriggers` is enough to either error out on user timestamps, or preserve them. Partially linearizable batches is just wonky and not really worth what this amount of code achieve that needs to be replaced anyways?